crates/rolldown_plugin_utils/src/file_to_url.rs

```diff
@@ -11,6 +11,15 @@ use crate::{PublicFileToBuiltUrlEnv, remove_special_query};
 use super::check_public_file::check_public_file;
 use super::find_special_query::find_special_query;
 
+const GIT_LFS_PREFIX: &[u8; 34] = b"version https://git-lfs.github.com";
+
+fn is_git_lfs_placeholder(content: &[u8]) -> bool {
+  if content.len() < GIT_LFS_PREFIX.len() {
```

Comment:
This seems redundenat, since rust lib would do it under the hood


```rs
// Generic slice equality
impl<A, B> SlicePartialEq<B> for [A]
where
    A: PartialEq<B>,
{
    default fn equal(&self, other: &[B]) -> bool {
        if self.len() != other.len() {
            return false;
        }

        // Implemented as explicit indexing rather
        // than zipped iterators for performance reasons.
        // See PR https://github.com/rust-lang/rust/pull/116846
        for idx in 0..self.len() {
            // bound checks are optimized away
            if self[idx] != other[idx] {
                return false;
            }
        }

        true
    }
}
```